charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -84,20 +88,80 @@ data:
     includes:
       - dynamic-plugins.default.yaml
     plugins:
+{{- if ne .Values.rhdhOperator.k8s.clusterToken "" }}
+      - package: ./dynamic-plugins/dist/backstage-plugin-kubernetes-backend-dynamic
+        disabled: false
+        pluginConfig:
+          kubernetes:
+            customResources:
+              - group: 'tekton.dev'
+                apiVersion: 'v1beta1'
```

Comment:
with recent OpenShift Pipeline operators, this should be `v1`.
We should be able to add both, to avoid any issues with it.

Reply:
Add a Tekton version to values.yaml?